java/ql/lib/semmle/code/java/frameworks/apache/IOGenerated.qll

```diff
@@ -1,684 +1,828 @@
-        "org.apache.commons.io;IOUtils;true;toByteArray;(URL);;Argument[0];open-url",
-        "org.apache.commons.io;IOUtils;true;toString;(URI);;Argument[0];open-url",
-        "org.apache.commons.io;IOUtils;true;toString;(URI,Charset);;Argument[0];open-url",
-        "org.apache.commons.io;IOUtils;true;toString;(URI,String);;Argument[0];open-url",
-        "org.apache.commons.io;IOUtils;true;toString;(URL);;Argument[0];open-url",
-        "org.apache.commons.io;IOUtils;true;toString;(URL,Charset);;Argument[0];open-url",
-        "org.apache.commons.io;IOUtils;true;toString;(URL,String);;Argument[0];open-url",
-        "org.apache.commons.io;RandomAccessFileMode;false;create;(File);;Argument[0];create-file",
-        "org.apache.commons.io;RandomAccessFileMode;false;create;(Path);;Argument[0];create-file",
-        "org.apache.commons.io;RandomAccessFileMode;false;create;(String);;Argument[0];create-file"
+        "org.apache.commons.io.file;PathFilter;true;accept;(Path,BasicFileAttributes);;Argument[-1];generated:regex-use[0]",
+        "org.apache.commons.io.file;PathFilter;true;accept;(Path,BasicFileAttributes);;Argument[0];generated:create-file",
+        "org.apache.commons.io.file;PathUtils;false;copyFile;(URL,Path,CopyOption[]);;Argument[0];generated:open-url",
+        "org.apache.commons.io.file;PathUtils;false;copyFile;(URL,Path,CopyOption[]);;Argument[1];generated:create-file",
+        "org.apache.commons.io.file;PathUtils;false;copyFileToDirectory;(URL,Path,CopyOption[]);;Argument[0];generated:open-url",
+        "org.apache.commons.io.file;PathUtils;false;newOutputStream;(Path,boolean);;Argument[0];generated:create-file",
+        "org.apache.commons.io.file;PathUtils;false;writeString;(Path,CharSequence,Charset,OpenOption[]);;Argument[0];generated:create-file",
+        "org.apache.commons.io.filefilter;FileFilterUtils;true;filter;(IOFileFilter,File[]);;Argument[1];generated:create-file",
+        "org.apache.commons.io.filefilter;FileFilterUtils;true;filterList;(IOFileFilter,File[]);;Argument[1];generated:create-file",
+        "org.apache.commons.io.filefilter;FileFilterUtils;true;filterSet;(IOFileFilter,File[]);;Argument[1];generated:create-file",
+        "org.apache.commons.io.filefilter;RegexFileFilter;true;RegexFileFilter;(String);;Argument[0];generated:regex-use[]",
+        "org.apache.commons.io.filefilter;RegexFileFilter;true;RegexFileFilter;(String,IOCase);;Argument[0];generated:regex-use[]",
+        "org.apache.commons.io.filefilter;RegexFileFilter;true;RegexFileFilter;(String,int);;Argument[0];generated:regex-use[]",
+        "org.apache.commons.io.input;Tailer$Tailable;true;getRandomAccess;(String);;Argument[-1];generated:create-file",
+        "org.apache.commons.io.input;XmlStreamReader;true;XmlStreamReader;(URL);;Argument[0];generated:open-url",
+        "org.apache.commons.io.output;DeferredFileOutputStream;true;writeTo;(OutputStream);;Argument[0];generated:create-file",
```

Comment:
create-file on OutputStream doesn't make much sense -- this should have been caught when the stream was created.